apps/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "devDependencies": {
     "@cdo/apps": "file:src",
-    "@code-dot-org/piskel": "0.2.0",
+    "@code-dot-org/piskel": "0.3.1",
```

Comment:
If interested, [here's the set of changes in this version bump](https://github.com/code-dot-org/piskel/compare/v0.2.0...v0.3.1).  The API has been made generic, and now supports a ready event, a save event, and a spritesheet load callback, as well as adding framerate support to those methods.